en/docs/learn/accessing-the-analytics-dashboard.md

```diff
@@ -67,12 +67,12 @@ Follow the steps below to access the WSO2 IS Analytics Dashboard.
 Explore the following topics:
 
 -   [Analyzing Overall Login
-    Attempts](Analyzing_Overall_Login_Attempts)
+    Attempts](analyzing-overall-login-attempts.md)
```

Comment:
need to give the relative path to the file. it should be in this format:
`(../../learn/analyzing-overall-login-attempts)`

please fix in all the relevant places

Reply:
Do I still have to add the relative path if both the files are residing in the same folder?

Reply:
yes